CHANGES.rst

```diff
@@ -27,6 +27,11 @@ extract_2d
 
 - Fix slice limits used in extraction of WFSS 2D cutouts. [#7312]
 
+general
+-------
+
+- Fix deprecation warnings introduced by ``pytest`` ``7.2`` ahead of ``8.0`` [#7325]
+
```

Comment:
Probably don't need a change log entry because this is just test infrastructure.  Not user-facing.